### ISSUE TITLE
Add work to receive_all() in wallet.py

### ISSUE THREAD

**Issue by elementalennui-dev** [jetstream0/bananopie#6]
Not a blocking issue, but work should be added to the receive_all() method so that each individual block can have custom work generated in receive_specific().

This worked for me locally:

```python
def receive_all(self, count=20, threshold=None, work = False):
    responses = []
    receivable_blocks = self.get_receivable_whole_threshold(count=count, threshold=threshold)["blocks"]
    for block_hash in receivable_blocks:
      #receive them
      responses.append(self.receive_specific(block_hash, work=work))
    return responses
```

**Comment by stjet** [comment]
Want to make a PR?

**Comment by elementalennui-dev** [comment]
Yeah, thanks!

**Comment by elementalennui-dev** [comment]
Closed by #7
